---
title: C6103 | Microsoft Docs
ms.custom: ''
ms.date: 2018-06-30
ms.prod: visual-studio-dev14
ms.reviewer: ''
ms.suite: ''
ms.technology:
- vs-devops-test
ms.tgt_pltfrm: ''
ms.topic: article
f1_keywords:
- C6103
ms.assetid: 22d1ab35-31a3-4ba9-8ef4-7a64bce66621
caps.latest.revision: 6
author: corob-msft
ms.author: gewarren
manager: ghogen
ms.openlocfilehash: bb1d8688af4c07fd2b0ee6fe67a56d1387610ff7
ms.sourcegitcommit: 55f7ce2d5d2e458e35c45787f1935b237ee5c9f8
ms.translationtype: MT
ms.contentlocale: pt-BR
ms.lasthandoff: 08/22/2018
ms.locfileid: "47462936"
---
# <a name="c6103"></a>C6103
[!INCLUDE[vs2017banner](../includes/vs2017banner.md)]

A versão mais recente deste tópico pode ser encontrada em [C6103](https://docs.microsoft.com/visualstudio/code-quality/c6103).  
  
Aviso C6103: retornando \<variável > de falha na chamada de função na linha \<local >.  
  
 Um caminho bem-sucedido por meio da função está retornando uma variável usada como um parâmetro `_Out_` para uma chamada de função interna malsucedida.  
  
 O problema talvez seja a função chamada e a função de chamada não anotadas por completo. A função de chamada pode exigir `_Always_`, `_Outptr_result_nullonfailure_` (`_COM_Outptr_` para código COM), ou uma anotação relacionada, e a função de chamada pode exigir uma anotação `_Success_`. Outra causa possível para esse aviso é que a anotação `_Success_` na função chamada está incorreta.  
  
## <a name="see-also"></a>Consulte também  
 [Usando anotações de SAL para reduzir defeitos de código do C/C++](../code-quality/using-sal-annotations-to-reduce-c-cpp-code-defects.md)



